src/ngx_http_lua_util.c

```diff
@@ -3880,6 +3888,18 @@ ngx_http_lua_init_vm(lua_State *parent_vm, ngx_cycle_t *cycle,
         lua_pop(L, 2);
     }
 
+    if (lmcf->load_resty_core) {
+        lua_getglobal(L, "require");
+        lua_pushstring(L, "resty.core");
+
+        rc = lua_pcall(L, 1, 1, 0);
+        if (rc != 0) {
+            ngx_log_error(NGX_LOG_ERR, log, 0,
+                          "lua_load_resty_core failed to load resty.core "
+                          "(%i): %s", rc, lua_tostring(L, -1));
```

Comment:
Better give more details here like recommending the use of OpenResty (with link) and the URL of the lua-resty-core library.

Reply:
Precisions added in the warning log